packages/devtools_app/lib/src/primitives/flutter_widgets/linked_scroll_controller.dart

```diff
@@ -372,14 +370,14 @@ class _LinkedScrollActivity extends ScrollActivity {
 
   void _updateUserScrollDirection() {
     assert(drivers.isNotEmpty);
-    ScrollDirection commonDirection;
+    ScrollDirection? commonDirection;
```

Comment:
this looks like it can be late instead of nullable

Reply:
It cannot because the line "commonDirection ??= driver.userScrollDirection" first read and then initializes.